### ISSUE TITLE
Possible bug / unexpected behaviour inside OccupancyGrid

### ISSUE THREAD

**Issue by stringnotfound1** [rosjava/android_core#297]
I'm currently working with the visualization part of android_core and wondering about the following [part ](https://github.com/rosjava/android_core/blob/kinetic/android_core_components/src/org/ros/android/view/visualization/layer/OccupancyGridLayer.java#L194) inside OccupancyGridLayer:

`        if (x < numTilesWide - 1) {
          tiles.get(tileIndex).setStride(TextureBitmap.STRIDE);
        } else {
          tiles.get(tileIndex).setStride(width % TextureBitmap.STRIDE);
        }`

I'm receiving a map with a 2048x2048 resolution and therefore 4 tiles are created. The tiles on top (x = 1) get a stride (aka width) of 0 assigned which makes the content invisible since the TextureBitMap [assigns the fillcolor](https://github.com/rosjava/android_core/blob/kinetic/android_core_components/src/org/ros/android/view/visualization/TextureBitmap.java#L114). Setting the stride of the tiles to TextureBitMap.STRIDE instead seems to solve the problem.

I don't know if the current implementation is solving a problem that I'm not aware of which is why I created this issue.

**Comment by jubeira** [comment]
Hi @stringnotfound1,
You might be right about this. If you send a PR with a fix, please include a couple of screenshots showing the before and after so that it's a bit easier to review.
Thanks in advance!

**Comment by stringnotfound1** [comment]
It might take a while until I manage to create the PR so here's a preview. All I did was replacing:
```java
if (x < numTilesWide - 1) {
    tiles.get(tileIndex).setStride(TextureBitmap.STRIDE);
} else {
    tiles.get(tileIndex).setStride(width % TextureBitmap.STRIDE);
}
```
with 
````java
tiles.get(tileIndex).setStride(TextureBitmap.STRIDE);
````
inside OccupancyGridLayer.

Before (I changed the COLOR_TRANSPARENT  to 0x55FF0000 for better visibility of the 'error'):

![occupancy_error_0x55FF0000](https://user-images.githubusercontent.com/32774990/55731445-8778c800-5a1a-11e9-8eab-52219e85e580.png)

After

![proper_occupancy_map](https://user-images.githubusercontent.com/32774990/55731476-96f81100-5a1a-11e9-9cb4-c2f444e119e2.png)

**Comment by jubeira** [comment]
Closed via #298; thanks again @stringnotfound1!
